app/models/bid_status_presenter_factory.rb

```diff
@@ -100,23 +100,23 @@ def ineligible?
   end
 
   def work_in_progress?
```

Comment:
It almost seems like this should be an enum on `delivery_status` -- perhaps we can create an issue to do that l8r?